woocommerce/pinterest-for-woocommerce
# Bugfix/handle false on api save

Closes #49 - hopefully 

### Changes proposed in this pull request

The `false` that the options API returns when there is no DB update needed is ignored.
If there is an error thrown by `apiFetch()` it ends up triggering the "Could not save settings" notice. 

#### Screenshots

![capture_YYYY07DD1607SS](https://user-images.githubusercontent.com/4016167/127503033-f675b32d-3851-44bc-9f1c-bbdbac02e2fa.gif)


### Detailed test instructions


1. Try clicking Save settings before changing any setting.
2. Confirm that you don't get an error message in case no setting was changed.
3. Confirm that after changing a setting and clicking "save settings" the change is transfered to the db.
